docs/electronics/marlin-firmware.md

```diff
@@ -1,205 +1,324 @@
-# Firmware
-Firmware is the program or software “embedded/burned/flashed” into the electronics board, this is what
-translates the gcode into motion in the stepper motors. There are many types of firmware we mainly use Marlin.
+<script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
 
-## Marlin
+# Marlin Firmware
 
-![!pic](https://www.v1engineering.com/wp-content/uploads/2015/05/Marlin-Logo-GitHub.png)
+This is where you can download the latest configured Marlin Firmware for your V1Engineering machines.
+Release notes and details are [in github](https://github.com/V1EngineeringInc/MarlinBuilder/releases)
 
-The boards sold here are using the [Marlin](https://github.com/MarlinFirmware/Marlin) Firmware, only
-minor changes needed to run any machine. Again, if you bought your electronics from here this has
-already been done for you initially, you can use this document to change or update your control board.
+After downloading the right firmware below, you can flash it to your controller ([Rambo, Mini-Rambo,
```

Comment:
This diff is very confusing. But the main thing I did here was add this line ("after downloading...") to the Marlin Releases page, and then I renamed it to marlin-firmware.md. 

So when someone clicks on marling firmware, they will be brought here. The first thing they will see (hopefully) is this statement, which will point them to how to flash.

After we merge this, we will have to change the link in the forums for MarlinBuilder releases to point to the new page.